fastparse/shared/src/main/scala/fastparse/sequence/VectorReprOps.scala

```diff
@@ -0,0 +1,34 @@
+package fastparse.sequence
+import fastparse.utils._
+import scala.reflect.ClassTag
+
+case class VectorReprOps[T]()(implicit evidence: ClassTag[T]) extends fastparse.utils.ReprOps[T, Vector[T]] {
+  type U = Vector[T]
+  val U = Vector
```

Comment:
This `val` does not appear to be used much; we should probably remove it and directly use `Vector` unless it's common enough to justify it's existence